AnnoyancesFilter/MobileApp/sections/mobile-app_specific.txt

```diff
@@ -9,6 +9,7 @@
 !
 ! SECTION: Mobile apps - Regular rules
 !
+tokfm.pl##.app-top-popup
```

Comment:
For apps banners on mobile devices only, we have a special section
https://github.com/AdguardTeam/AdguardFilters/blob/29e5df723869778c43e7efe388670e8fc6cc385c/AnnoyancesFilter/MobileApp/sections/mobile-app_specific.txt#L2080